cranelift/codegen/Cargo.toml

```diff
@@ -61,14 +64,16 @@ arm32 = []
 arm64 = []
 riscv = []
 x64 = [] # New work-in-progress codegen backend for x86_64 based on the new isel.
+spirv = []
 
 # Option to enable all architectures.
 all-arch = [
     "x86",
     "x64",
     "arm32",
     "arm64",
-    "riscv"
+    "riscv",
+    "spirv"
```

Comment:
Does toml support trailing commas? If so, please add one to minimize diffs a bit for future architecture additions.

Reply:
Seems so, fixed.